src/components/EmojiPicker/EmojiPickerMenu/getItemType.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
We don't allow new JavaScript files, this should be ported to TypeScript

Reply:
[This](https://expensify.slack.com/archives/C01GTK53T8Q/p1700075112873269) announcement doesn't mention `src/components`. Should I do that anyway? It would require typing the `item`